packages/react-magma-dom/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -22,19 +23,38 @@ export interface NativeSelectProps
   /**
    * @internal
    */
-  testId?: string;
   optionLabel?: string;
+  testId?: string;
 }
-const StyledNativeSelect = styled.select<{
+const StyledNativeSelectWrapper = styled.div<{
+  disabled?: boolean;
+  hasError?: boolean;
+  isInverse?: boolean;
   theme: ThemeInterface;
+}>`
+  align-items: center;
+  display: flex;
```

Comment:
These two styles are already in `inputWrapperStyles`. 

I would suggest having `inputWrapperStyles` first, and then below any new style that is missing or needs to be overwritten.